src/main/java/santiagoAndFerdy/vgs/resourceManager/ResourceManagerGridScheduleClient.java

```diff
@@ -0,0 +1,78 @@
+package santiagoAndFerdy.vgs.resourceManager;
```

Comment:
Also just for testing right? Seems like there's a lot of duplication in our GS and RM classes. Can be reduced by a lot by injecting the schedulers and if you polish up the heartbeat classes a bit.